src/test/kotlin/day15/repository/CommentRepositoryImplTest.kt

```
@@ -0,0 +1,10 @@
+package day15.repository
+
+import org.junit.Test
+
+import org.junit.Assert.*
+ class CommentRepositoryImplTest {
+
+@Test
+ fun getComments() {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)